pages/projects/[id]/index.js

```diff
@@ -0,0 +1,32 @@
+import { useRouter } from "next/router";
+import useSWR from "swr";
+
+import Error from "@/components/Error";
+import Heading from "@/components/Heading";
+import Button from "@/components/Button";
+
+export default function ProjectDetailPage() {
+  const router = useRouter();
+  const { isReady } = router;
+  const { id } = router.query;
+
+  const { data: project, isLoading, error } = useSWR(`/api/projects/${id}`);
+
+  if (!isReady || isLoading) return <h2>Loading..</h2>;
+  if (error) return <Error message={error.message} />;
+
+  const { title, description, endDate, department, teamLead } = project;
+
+  return (
+    <>
+      <Button />
+      <Heading>{title}</Heading>
```

Comment:
here again, you are rendering h1 heading for every porject. Your h1 should be your app title or main heading. here you should work with h2 (make sure to not skip any headline)